core/trino-parser/src/main/java/io/trino/sql/tree/DefaultTraversalVisitor.java

```diff
@@ -636,6 +636,17 @@ protected Void visitUpdateAssignment(UpdateAssignment node, C context)
         return null;
     }
 
+    @Override
+    protected Void visitMerge(Merge node, C context)
```

Comment:
Isn't there a need to implement merge cases too?

Reply:
Indeed there is.  Implemented.